src/js/component.js

```diff
@@ -115,6 +115,8 @@ class Component {
     if (options.reportTouchActivity !== false) {
       this.enableTouchActivity();
     }
+
+    this.handleLanguageChange = this.handleLanguageChange.bind(this);
```

Comment:
This is around where the change I mentioned in clickable component should live. I don't think we will need the bind anymore either.